src/client.js

```diff
@@ -37,11 +37,16 @@ export default class Client {
     this.api = {};
 
     this.compose = {
-      async message(app, payload, wif) {
-        const privKeyBuf = wifLib.decode(wif, self.testnet ? 239 : 128).privateKey;
+      async message(app, payload, options = {}) {
+        const conf =
+          typeof options === 'object'
+            ? { ...self.options, ...options }
+            : { ...self.options, ...{ wif: options } };
```

Comment:
nit: You don't have to spread second object, you can just do this:

```suggestion
            : { ...self.options, wif: options };
```